<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-FL7P4KD" failures="1" tests="1" name="com.purna.runner.RunPurnaTest" time="10.930" errors="0" timestamp="2023-08-31T11:10:41 IST" skipped="0">
  <testcase classname="com.purna.runner.RunPurnaTest" name="runScenario" time="10.930">
    <failure message="expected:&lt;3&gt; but was:&lt;0&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<3> but was:<0>
at org.junit.Assert.fail(Assert.java:93)
at org.junit.Assert.failNotEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:128)
at org.junit.Assert.assertEquals(Assert.java:472)
at org.junit.Assert.assertEquals(Assert.java:456)
at com.purna.stepdefinitions.DeleteSupplier_stepdefinition.user_verifies_that_the_supplier_details_is_deleted(DeleteSupplier_stepdefinition.java:49)
at âœ½.User verifies that the supplier details is deleted(file:///C:/Drivers/web-ui-bdd-framework/web-ui-bdd-framework/src/test/resources/features/DeleteSuplier.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.purna.runner.RunPurnaTest -->
